lesson35/lesson35_part2.py

```diff
@@ -14,7 +14,6 @@ def car_info(self):
         method returns car info
         :return: string of car info
         """
-
         return (f'{self.model} {self.year}:'
                 f'\nEngine - {self.engine}'
                 f'\nMileage - {self.mileage} км'
```

Comment:
def add_car
This method doesn't have any difference from constructor, so doesn't make any sense